src/modules/cmdpal/WindowsCommandPalette/Views/MainViewModel.xaml.cs

```diff
@@ -162,5 +164,6 @@ private void PopulateAliases()
         this.AddAlias(new CommandAlias("$", "com.microsoft.cmdpal.windowsSettings", true));
         this.AddAlias(new CommandAlias("=", "com.microsoft.cmdpal.calculator", true));
         this.AddAlias(new CommandAlias(">", "com.microsoft.cmdpal.shell", true));
+        this.AddAlias(new CommandAlias("??", "com.microsoft.cmdpal.websearch", true));
```

Comment:
I was coming to suggest it'd be cool if `?` was the alias for websearch, as that's the common search key for the addressbar (at least in Edge). So, why is there two `??` here?

Reply:
This is solely a vestige of how it works as the default today in PT Run. I am merely a conduit at the moment getting things migrated in, but I would be very open to revisiting this.